packages/bits/spec/virtual-camera/EyesLens.ts

```diff
@@ -4,7 +4,7 @@ import { getCurrentBranchName } from "../helpers";
 export class EyesLens implements ILens {
```

Comment:
I just noticed that the filename doesn't follow the kebab-case naming convention